### DISCUSSION TITLE
Multi Range

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [saadeghi/daisyui#2079] [Q&A] [answered]
Hi,

Is it possible to have a two ranges in the slider of
`<input type="range" min="0" max="100" value="40" class="range" />`

Similar like this

![Screenshot from 2023-06-30 10-30-15](https://github.com/saadeghi/daisyui/assets/22890183/b1073269-bc8c-4e0a-b39d-56ffceee664f)

**Comment by saadeghi** [comment, marked as answer]
See this https://github.com/saadeghi/daisyui/issues/573#issuecomment-1064453159
